le.sh

```diff
@@ -279,6 +279,9 @@ _initpath() {
   if [ -z "$CA_CERT_PATH" ] ; then
     CA_CERT_PATH="$WORKING_DIR/$domain/ca.cer"
   fi
+  if [ -z "$CERT_CHAIN_PATH" ] ; then
```

Comment:
Sure - I mainly this this pull request because of https://github.com/Neilpang/le/issues/24#issuecomment-172228674